doc/howto/network_bridge_firewalld.md

```diff
@@ -1,52 +1,120 @@
-(network-bridge-firewalld)=
-# How to configure Firewalld
+---
+discourse: 10034,9953
+---
 
-## Allow DHCP, DNS with Firewalld
+(network-bridge-firewall)=
+# How to configure your firewall
 
-In order to allow instances to access the DHCP and DNS server that LXD runs on the host when using firewalld
-you need to add the host's bridge interface to the `trusted` zone in firewalld.
+If you run a firewall on your system, you might need to configure it to allow network traffic between the managed LXD bridge and the host.
+Otherwise, some network functionality (DHCP, DNS and external network access) might not work as expected.
 
-To do this permanently (so that it persists after a reboot) run the following command:
+By default, managed LXD bridges add firewall rules to ensure full functionality.
+(This behavior can be controlled through the `ipv4.firewall` or `ipv6.firewall` {ref}`configuration options <network-bridge-options>`.)
+However, depending on the firewall subsystem that you use, firewall rules added by other applications might interfere with the firewall rules that LXD adds.
 
-```
-firewall-cmd --zone=trusted --change-interface=<LXD network name> --permanent
-```
+See the following sections for instructions on how to properly configure firewalld and UFW, respectively.
 
-E.g. for a bridged network called `lxdbr0` run the command:
+## Allow traffic from/to the LXD bridge
 
-```
-firewall-cmd --zone=trusted --change-interface=lxdbr0 --permanent
-```
+To allow instances to access the DHCP and DNS server that LXD runs on the host, you must configure your firewall to allow network traffic from and to the LXD bridge.
+This will then allow LXD's own firewall rules to take effect for all instances that use the network bridge.
 
-This will then allow LXD's own firewall rules to take effect.
+### Firewalld: Add the bridge to the trusted zone
 
+To allow traffic from and to the LXD bridge in firewalld, add the bridge interface to the `trusted` zone.
+To do this permanently (so that it persists after a reboot), run the following command:
 
-## How to let Firewalld control the LXD's iptables rules
+    firewall-cmd --zone=trusted --change-interface=<network_bridge> --permanent
 
-When using firewalld and LXD together, iptables rules can overlaps. For example, firewalld could erase LXD iptables rules if it is started after LXD daemon, then LXD container will not be able to do any oubound internet access.
-One way to fix it is to delegate to firewalld the LXD's iptables rules and to disable the LXD ones.
+For example:
 
-First step is to [allow DNS and DHCP](#allow-dhcp-dns-with-firewalld).
+    firewall-cmd --zone=trusted --change-interface=lxdbr0 --permanent
```

Comment:
These 2 commands suffice without needing the `--direct` rules below.
However please can you prefix them with "sudo", and add ` sudo firewall-cmd --reload` at the end so they are applied.